Documentation to support GCP for Chef Automate HA (#8214)
chef/automate: components/docs-chef-io/content/automate/ha_config_gen.md

## Backup

> - If configuring backup at the time of deployment the  we many need following fields:

Reviewer: ```suggestion
- If configuring backup at the time of deployment the we may need following fields:
```

Author: done

>   - `Bucket name` for object storage or Google Cloud Storage bucket name in google cloud platform or AWS S3 type of backup provide bucket name or , for AWS deployment bucket will be created if not exist in AWS.

Reviewer: ```suggestion
  - `bucket_name` for object storage or Google Cloud Storage bucket name in google cloud platform or AWS S3 type of backup provide bucket name or , for AWS deployment bucket will be created if not exist in AWS.
```
The heading introductory line of the section says, "fields". But "Bucket name" is a prompt, and the storage type it shows are "options" Can we modify the tag as it is confusing now?

Author: updated the introductory line

  - `Access Key` S3 access key.
  - `Secret Key` S3 secret key.
  - `Endpoint` for object storage provide endpoint of object storage.

>   - `Region` for S3 provide region.

Reviewer: ```suggestion
  - `Region` S3 bucket region.
```

>   - `Endpoint` for object storage provide endpoint of object storage.
>   - `Region` for S3 provide region.
>   - `Mount path` in case of file system/efs backup provide mount path of backup directory.

Reviewer: ```suggestion
  - `Endpoint` Endpoint of object storage.
  - `Region` S3 bucket region.
  - `Mount path` in case of file system/efs backup, provide the mount path of backup directory.
```

Author: done
